bundles/org.openhab.binding.dwdpollenflug/src/main/java/org/openhab/binding/dwdpollenflug/internal/handler/DWDPollenflugRegionHandler.java

```diff
@@ -0,0 +1,127 @@
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.dwdpollenflug.internal.config.DWDPollenflugRegionConfiguration;
+import org.openhab.binding.dwdpollenflug.internal.dto.DWDPollenflug;
+import org.openhab.binding.dwdpollenflug.internal.dto.DWDRegion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DWDPollenflugRegionHandler} is the handler for bridge thing
+ *
+ * @author Johannes DerOetzi Ott - Initial contribution
+ */
+@NonNullByDefault
+public class DWDPollenflugRegionHandler extends BaseThingHandler implements DWDPollenflugRegionListener {
+
+    private final Logger logger = LoggerFactory.getLogger(DWDPollenflugRegionHandler.class);
+
+    private DWDPollenflugRegionConfiguration thingConfig = new DWDPollenflugRegionConfiguration();
+
+    public DWDPollenflugRegionHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing DWD Pollenflug region handler");
+        thingConfig = getConfigAs(DWDPollenflugRegionConfiguration.class);
+
+        if (thingConfig.isValid()) {
+            DWDPollenflugBridgeHandler handler = syncToBridge();
+            if (handler == null) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Bridge handler missing");
+            } else {
+                handler.registerRegionListener(this);
+                updateStatus(ThingStatus.ONLINE);
+            }
+        } else {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR);
+        }
+    }
+
+    private synchronized @Nullable DWDPollenflugBridgeHandler syncToBridge() {
+        Bridge bridge = getBridge();
+        if (bridge != null) {
+            ThingHandler handler = bridge.getHandler();
+            if (handler instanceof DWDPollenflugBridgeHandler) {
+                DWDPollenflugBridgeHandler bridgeHandler = (DWDPollenflugBridgeHandler) handler;
+                return bridgeHandler;
+            }
+        }
+
+        return null;
+    }
+
+    @Override
+    public void dispose() {
+        logger.debug("DWDPollenflug region handler disposes. Unregistering listener.");
+        DWDPollenflugBridgeHandler bridgeHandler = syncToBridge();
+        if (bridgeHandler != null) {
+            bridgeHandler.unregisterRegionListener(this);
+        }
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            refresh();
+        }
+    }
+
+    @Override
+    public void channelLinked(ChannelUID channelUID) {

```

Comment:
Not needed. A `REFRESH` command should be sent when a channel is linked.

Reply:
Removed